modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/home/UserActivity.es.js

```diff
@@ -59,9 +61,27 @@ export default withRouter(
 			document.title = creatorId;
 		}, [creatorId]);
 
-		const {data, loading} = useQuery(getUserActivityQuery, {
-			onCompleted(data) {
-				if (data.messageBoardMessages.items.lenght) {
+		const [fetchUserActivity, {data}] = useManualQuery(
+			getUserActivityQuery,
+			{
+				variables: {
+					filter: `creatorId eq ${creatorId}`,
+					page,
+					pageSize,
+					siteKey,
+				},
+			}
+		);
+
+		useEffect(() => {
+			if (!page || !pageSize) {
```

Comment:
but what happens if they are no parameters?

Reply:
I had this doubt and I've checked that the queryParams hooks are always executed once to put the default value

Reply:
But what are the defaults for the queryParams?

Reply:
The hook is always called at least once and fall in the default used in the hook, 1 for page, 20 for pageSize, and '' for search

Reply:
Ok, I see they get it from the useEffect that is called when queryParams has something.